index.html

```diff
@@ -0,0 +1,186 @@
+    <link rel="stylesheet" href="teachers.css">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&family=Lato&family=Poppins:wght@300&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Skill-Sail College</title>
+</head>
+<body>
+    <header class="primary-nav desk-version">
+        <nav class="header-nav">
+            <ul>
+                <li><a href="facebook.com" class="nav-links">
+                    <i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="twitter.com" class="nav-links">
+                    <i class="fa-brands fa-facebook"></i></a></li>
+                <li><a href="#english" class="nav-links">English</a></li>
+                <li><a href="#mypage" class="nav-links">Mypage</a></li>
+                <li><a href="#logout" class="nav-links">Logout</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class="main">
+        <header class="secodary-nav">
+            <div class="logo">
+                <a href="index.html">
+                   <i class="fa-solid fa-sailboat">  GENIUS</i> <span>ACADEMY</span></a>
+            </div> 
+            <nav class="nav-bar">
+                <ul>
+                    <li><a href="about.html" class="nav-links about">About</a></li>
+                    <li><a href="#programms" class="nav-links">Programms</a></li>
+                    <li><a href="#featuredteacher" class="nav-links">Teachers</a></li>
+                    <li><a href="#contact" class="nav-links">Join</a></li>
+                    <button type="button" class="nav-btn">Genius boost</button>
+                </ul>               
+            </nav>
+            <div class="img-nav">
+                <a href="#"><i class="fa-solid fa-bars humberger"></i> 
+                    <i class="fa-solid fa-xmark close"></i>
+                </a>  
+            </div>
+        </header>
+
+<section class="introduction-page">
+    <div class="intro-text">
+        <div class="inside">Hello! student.</div>
+        <h1>
+            <span>Welcome to </span> <br> Genius Academy, Sail it to your dreams<h1>
+    </div>
+        <p class="intro-par">Join the Genius Academy community and start sailing towards a brighter future today!</p>
+    <div class="time">
+        <p>Open from Saturday-Wednesday on 8:00 A.M  ~ 5:30 P.M</p>
+        <p class="location"> @ Saiberia Street near by Masjid Harawa</p>
+    </div>
+</section>
+</section>
+
+<main class="programms" id="programms">
```

Comment:
- [ ] [OPTIONAL] Good so far :clap: But I think you can improve and I would strongly like to recommend you to make your program section follow the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: Colors, Layout: composition and space between elements to make your website look wonderful :+1: